src/Integration.UnitTests/Suppression/SonarQubeIssuesProviderTests.cs

```diff
@@ -104,33 +106,17 @@ public void Constructor_TimerIsInitialized()
             VerifyTimerStart(Times.Once());
             timerRunning.Should().Be(true);
 
-            // The call to service.IsConnected is on a background thread so we can't
-            // reliably check whether it has occurred yet. However, we know that 
-            // service.GetIssues(...) should never have been called since the service
-            // is not connected.
-            VerifyServiceGetIssues(Times.Never());
+            WaitForInitialFetchTaskToStart();
+            VerifyServiceIsConnected(Times.Exactly(2));
```

Comment:
Updated existing test: changed the expected call counts to take account of the fact that the initial fetch has completed.